main.tf

```diff
@@ -129,5 +148,33 @@ resource "aws_cloudwatch_event_target" "security_reporter_rule_target" {
   arn       = aws_lambda_function.security_reporter_lambda.arn
 }
 
+resource "aws_ssm_parameter" "security_controls_api_host" {
+  name  = local.ps_key_security_controls_api_host
```

Comment:
I know that you already had variables for keys and values. I think we could stay with variables with a default value just in case if somebody will have a need to override it. For example:

```
variable "ps_key_security_controls_api_host" {
  description = "Name of the SSM parameter where a host to REST API returning controls ID is stored"
  type = "String"
  default = "/security/hub/reporter/api/host"
}

environment {
    variables = {
            PS_KEY_CONTROLS_IDS_API_HOST     = var.ps_key_security_controls_api_host

resource "aws_ssm_parameter" "security_controls_api_host" {
  name  = var.ps_key_security_controls_api_host
  type  = "String"
  value = "NOT_SET"
  lifecycle {
    ignore_changes = [value]
  }
}


```